NuRadioReco/modules/correlationDirectionFitter.py

```diff
@@ -227,22 +228,24 @@ def ll_regular_station_fft(angles, corr_02_fft, corr_13_fft, sampling_rate, posi
 #             print("offset 13= {:.3f}".format(toffset[indices[imax]] -  (delta_t_13 / sampling_rate)))
 
             ax2.axvline(delta_t_13 / sampling_rate, label='time', c='k')
-            
+
             ax2.set_xlabel("time")
             ax2.set_ylabel("Correlation Ch 1/ Ch3", fontsize='small')
             ax.set_ylabel("Correlation Ch 0/ Ch2", fontsize='small')
             plt.tight_layout()
 #             plt.close("all")
 
-        station[stnp.zenith] = max(ZenLim[0], min(ZenLim[1], ll[0][0]))
-        station[stnp.azimuth] = ll[0][1]
-        output_str = "reconstucted angles theta = {:.1f}, phi = {:.1f}".format(station[stnp.zenith] / units.deg, station[stnp.azimuth] / units.deg)
-        if station.has_sim_station():
+        shower = evt.get_first_shower([station.get_id()])
+        shower.set_parameter(shp.zenith, max(ZenLim[0], min(ZenLim[1], ll[0][0])))
```

Comment:
this is wrong for neutrinos. The shower direction is not the incoming signal direction. The incoming signal direction should be saved in the station parameters (as before).